frontends/ol-search-ui/src/components/LearningResourceCard.tsx

```diff
@@ -121,9 +114,26 @@ const LearningResourceCard: React.FC<LearningResourceCardProps> = ({
           </span>
           {hasCertificate && <CertificateIcon />}
         </div>
-        <Dotdotdot className="ol-lrc-title" tagName="h3" clamp={3}>
-          {resource.title}
-        </Dotdotdot>
+        {toggleDrawer ? (
+          <div
+            className="clickable"
+            onClick={() =>
+              toggleDrawer({
+                type:  resource.object_type,
+                id:    resource.id,
+                runId: bestAvailableRun?.id
+              })
+            }
+          >
+            <Dotdotdot className="ol-lrc-title" tagName="h3" clamp={3}>
+              {resource.title}
+            </Dotdotdot>
+          </div>
```

Comment:
This should be clickable/focusable with the keyboard. Although we might want to check with Ferdi/peter whether the whole card should be clickable, or just the title.

If we do decide that just the title should be clickable, it would be nice to use a more semantic component